guides/features/pipecat-flows.mdx

```diff
@@ -1,441 +1,267 @@
+<CardGroup cols={2}>
   <Card
-    title="Movie Explorer (Static)"
-    icon="film"
-    href="https://github.com/pipecat-ai/pipecat-flows/blob/main/examples/static/movie_explorer_openai.py"
+    title="Hello World"
+    icon="rocket"
+    href="https://github.com/pipecat-ai/pipecat-flows/tree/main/examples/quickstart"
   >
-    A static flow demonstrating movie exploration using OpenAI. Shows real API
-    integration with TMDB, structured data collection, and state management.
+    A great first Flow to show you the ropes
   </Card>
-
   <Card
-    title="Insurance Policy (Dynamic)"
-    icon="shield"
-    href="https://github.com/pipecat-ai/pipecat-flows/blob/main/examples/dynamic/insurance_gemini.py"
+    title="Restaurant Reservation"
+    icon="utensils"
+    href="https://github.com/pipecat-ai/pipecat-flows/blob/main/examples/dynamic/restaurant_reservation.py"
   >
-    A dynamic flow using Google Gemini that adapts policy recommendations based
-    on user responses. Demonstrates runtime node creation and conditional paths.
+    A simple, practical dynamic flow example
   </Card>
 </CardGroup>
 
-<Note>
-  These examples are fully functional and can be run locally. Make sure you have
-  the required dependencies installed and API keys configured.
-</Note>
-
-## When to Use Static vs Dynamic Flows
-
-**Static Flows** are ideal when:
-
-- Conversation structure is known upfront
-- Paths follow predefined patterns
-- Flow can be fully configured in advance
-- Example: Customer service scripts, intake forms
-
-**Dynamic Flows** are better when:
-
-- Paths depend on external data
-- Flow structure needs runtime modification
-- Complex decision trees are involved
-- Example: Personalized recommendations, adaptive workflows
-
-# Installation
-
-If you're already using Pipecat:
-
-```bash
-pip install pipecat-ai-flows
-```
-
-If you're starting fresh:
-
-```bash
-# Basic installation
-pip install pipecat-ai-flows
-
-# Install Pipecat with specific LLM provider options:
-pip install "pipecat-ai[daily,openai,deepgram]"     # For OpenAI
-pip install "pipecat-ai[daily,anthropic,deepgram]"  # For Anthropic
-pip install "pipecat-ai[daily,google,deepgram]"     # For Google
-```
-
-💡 Want to design your flows visually? Try the [online Flow Editor](https://flows.pipecat.ai)
+## How do Pipecat and Pipecat Flows work together?
 
-# Core Concepts
+**Pipecat** defines the core capabilities of your bot. This includes the pipeline and processors which, at a minimum, enable your bot to:
 
-## Designing Conversation Flows
+- Receive audio from a user
+- Transcribe the user's input
+- Run an LLM completion
+- Convert the LLM response to audio
+- Send audio back to the user
 
-Functions in Pipecat Flows serve two key purposes:
+**Pipecat Flows** complements Pipecat's core functionality by managing the conversation's context and tools. This is separate from the core pipeline, allowing you to separate the conversation logic from the core pipeline mechanics.
```

Comment:
IMO, "managing the conversation's context and tools" kind "undersells" what the framework does. It doesn't say to what end it does those things.

Taking a stab at summarizing:

```suggestion
**Pipecat Flows** complements Pipecat's core functionality by providing a layer of structure to a conversation, managing context and tools as the conversation moves from one state to another. This is separate from the core pipeline, allowing you to separate conversation logic from core pipeline mechanics.
```